### ISSUE TITLE
Commit message from CommitPack unused?

### ISSUE THREAD

**Issue by SeanHeelan** [bigcode-project/octopack#23]
Hey folks, 

Nice work! Something came to mind as I browsed your code: it looks like you only used the commit subject from CommitPack during training of OctoCoder. Is that correct? (I'm concluding it based on what you have said in #9 regarding the format, and the contents of the repository [here](https://huggingface.co/datasets/bigcode/guanaco-commits)). 

Did you guys experiment with using the commit message as well? Or was there a reason you decided not to use it? 

Thanks!

**Comment by Muennighoff** [comment]
Yes that's correct. The reason is that the message is usually exactly the same. If it's not the same it often includes external references which we don't want.

You can browse samples here: https://huggingface.co/datasets/bigcode/commitpackft/viewer/kotlin?row=1
